### ISSUE TITLE
Inline drafts lost when new email arrives

### ISSUE THREAD

**Issue by callahad** [protz/thunderbird-conversations#1100]
STR:
1. Select a thread, click the inline "Reply All" on the latest message.
2. Begin composing a reply.
3. Have someone else reply to the same thread

What should happen:
1. The new message appears at the end of the list without interrupting or reflowing my current draft.

What actually happens:
1. The new message appears at the end of the list, collapsing my current message, which hides my draft and makes it unrecoverable.

Specifically, I cannot see the inline reply boxes anymore, so I can't see my draft there. It also does not appear in my Drafts folder, or my Trash folder.

I have not yet tested to see if this is reproducible. Other potential factors:
- My Thunderbird is set to save drafts to local folders.
- I was Cmd-Tabbing between applications, and Thunderbird was in the background when the new mail arrived.
- I'm using build of the extension from git master on May 26th.

**Comment by protz** [comment]
Yes... this is probably minor to fix, though. Right now, the draft is saved on the "blur" event, and probably we just need to save it on a "input" event.

**Comment by cyberDrake** [comment]
I do not know if it is related or if it should be a new bug, but the "save" button on the quick reply does nothing for me. I can no longer save any draft from quick reply - if I navigate to a different message, anything I have typed is lost.

**Comment by FlorentBecker** [comment]
I don't get why this is tagged minor. It happens quite systematically, and makes quick reply useless in any heated discussion (where you most want to use it).

**Comment by Standard8** [comment]
@cyberDrake I've separated it out to #1257.

@FlorentBecker I suspect that for most people this isn't too big of an issue, and so I agree with protz's determination for minor.

**Comment by rippetoej** [comment]
I've lost many emails this way and it is incredibly frustrating, so I wouldn't say this is a minor issue for me. But perhaps I am in the minority on this, but it would be nice to have this bug resolved.

**Comment by hsandt** [comment]
I think the bigger issue behind this is that the embedded editor (as opposed to the full popup HTML editor) doesn't save drafts at all:
- there is no auto-save
- if you focus something else you will lose the draft without a "discard?" prompt
- Ctrl+S will prompt for manual .eml saving in custom location, which is more cumbersome to use
